adapters/vrtcal/vrtcal.go

```diff
@@ -70,10 +70,13 @@ func (a *VrtcalAdapter) MakeBids(internalRequest *openrtb2.BidRequest, externalR
 
 	for _, sb := range bidResp.SeatBid {
 		for i := range sb.Bid {
-			bidResponse.Bids = append(bidResponse.Bids, &adapters.TypedBid{
-				Bid:     &sb.Bid[i],
-				BidType: "banner",
-			})
+			bidType, err := getReturnTypeForImp(sb.Bid[i].ImpID, internalRequest.Imp)
+			if err == nil {
```

Comment:
@vrtcal-dev This check would just ignore the error. I'd recommend changing this to the following so that the errors are properly communicated to the user.
```
if (err != nil) {
	errs = append(errs, err)
	continue
}
bidResponse.Bids = append(bidResponse.Bids, &adapters.TypedBid{
	Bid:     &sb.Bid[i],
	BidType: bidType,
})
```
You'll just. have to instantiate and `errs` slice either at the start of this function or before the for loop

Reply:
> This check would just ignore the error. I'd recommend changing this to the following so that the errors are properly communicated to the user.

Thanks mansinahar, I've udpated to properly report errored bids while continuing to proces further bids.